reporter/grpc/management/Management.proto

```diff
@@ -19,20 +19,28 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "org.apache.skywalking.apm.network.language.agent.v2";
+option java_package = "org.apache.skywalking.apm.network.management.v3";
 option csharp_namespace = "SkyWalking.NetworkProtocol";
-option go_package = "github.com/SkyAPM/go2sky/reporter/grpc/language-agent-v2";
+option go_package = "github.com/SkyAPM/go2sky/reporter/grpc/management";
 
+import "common/Common.proto";
 
-import "common/common.proto";
-import "common/JVM.proto";
+service ManagementService {
```

Comment:
Seems you haven't implemented these two? It could be an issue, if no traffic happens on the go app, the service/instance disappear.

Reply:
It has been implemented, [click here](https://github.com/arugal/go2sky/blob/a9a8a9cbd19ee3b100db316e1cc432aac969290b/reporter/grpc.go#L221-L266).